### PR TITLE
Added bool WasDisguised

### PR DESCRIPTION
**Pull request by NikkiGardiner1**
This has been tested.

.wasDisguised is used a bit throughout the 3114-identity class. Which controls if he can/cannot attack, if he is able to remove his disguise, and activates/devastates voicelines for 3114

### CODE REVIEW

**Comment by Misaka-ZeroTwo** [comment]
Please resolve merge conflict

**Comment by NikkiGardiner1** [comment]
> Please resolve merge conflict

Merge conflict resolved

File: Exiled.API/Features/Roles/Scp3114Role.cs [hunk 1]
```diff
@@ -217,12 +217,22 @@ public void ResetIdentity()
...
+            set => Identity._wasDisguised = value;
+        }
 
         /// <summary>
         /// Gets the Spawn Chance of SCP-3114.
         /// </summary>
         /// <param name="alreadySpawned">The List of Roles already spawned.</param>
         /// <returns>The Spawn Chance.</returns>
         public float GetSpawnChance(List<RoleTypeId> alreadySpawned) => Base is ISpawnableScp spawnableScp ? spawnableScp.GetSpawnChance(alreadySpawned) : 0;
+        halloween-2023
```
Comment (Misaka-ZeroTwo) [inline Exiled.API/Features/Roles/Scp3114Role.cs:236]:
```suggestion

```

Comment (Misaka-ZeroTwo) [reply Exiled.API/Features/Roles/Scp3114Role.cs:236]: You forgot this line

**Comment by NikkiGardiner1** [comment]
Should be good now, mb for missing it

File: Exiled.API/Features/Roles/Scp3114Role.cs [hunk 2]
```diff
@@ -217,6 +217,15 @@ public void ResetIdentity()
...
             Identity.ServerResendIdentity();
         }
+        
+        /// <summary>
+        /// Gets or sets wasDisguised boolean.
+        /// </summary>
+        public bool WasDisguised
+        {
+            get => Identity._wasDisguised;
+            set => Identity._wasDisguised = value;
```
Comment (louis1706) [inline Exiled.API/Features/Roles/Scp3114Role.cs:227]: Can you make sure than setter update with client (i actually thinks than it's don't and than it's will imediatelly be overided)

**Review by louis1706 — CHANGES_REQUESTED** [review]
Some Verification

**Pull request closed without merging**